### ISSUE TITLE
Developer forum needs specification

### ISSUE THREAD

**Issue by jtmelton** [jowasp/review-features#5]
A lot of people these days use stackoverflow or even the project mailing list as a forum. Is that acceptable? Do we need to maintain our own actual forum? If so, does the foundation provide tooling for this?

**Comment by stevespringett** [comment]
Agreed. Stack Overflow is popular, as is [mailman lists](lists.owasp.org), and Google Groups. This requirement needs further clarification.

**Comment by jowasp** [comment]
I have defined a basic description of this requirement:
https://github.com/jowasp/review-features/blob/master/developer.md
Please take a look.
